python/private/toolchain_defs.bzl

```diff
@@ -0,0 +1,81 @@
+# Copyright 2022 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Create the toolchain defs in a BUILD.bazel file."""
+
+load("@bazel_skylib//lib:selects.bzl", "selects")
+
+_py_toolchain_type = Label("@bazel_tools//tools/python:toolchain_type")
+_py_cc_toolchain_type = Label("//python/cc:toolchain_type")
+
+def toolchain_defs(*, prefix, user_repository_name, python_version, set_python_version_constraint, **kwargs):
```

Comment:
Perhaps `define_toolchain_suite()` instead? That a more verby-name and "suite" refers to how there's more than just a singe toolchain_type that is defined.

Reply:
Chose `py_toolchain_suite` as there was `cc_toolchain_suite` already.